Makefile

```diff
@@ -1,13 +1,24 @@
-package: 
+.PHONY: clippy dev format lint package run scratch test ts_test test_application test_rust test_rehash
+
+ifeq ($(OSTYPE),darwin)
+PACKAGE := application/release/mac/WereSoCool.app
+else
+PACKAGE := application/release/linux-unpacked/weresocool
+endif
```

Comment:
I think the `OSTYPE` variable will work for switching on OS-specific tasks. The `ifeq`, `else`, and `endif` lines are run in a pre-processor; imagine that make evaluates all the `ifeq`s first, removes all the lines except the ones in the matching conditional branch, and then evaluates the resulting Makefile.